hbase-server/pom.xml

```diff
@@ -524,6 +524,44 @@
       <plugin>
         <artifactId>maven-antrun-plugin</artifactId>
         <executions>
+          <execution>
+            <!-- Copy static CSS and JS files for the UI. -->
+            <id>copy-css-js-to-ui</id>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <phase>process-resources</phase>
+            <configuration>
+              <target>
+                <copy flatten="true" todir="${project.build.directory}/hbase-webapps/static/css">
```

Comment:
Is it okay to flatten these files? I doubt that we want to perform any processing on their content...

Reply:
I removed this whole step now.